bin/README.md

```diff
@@ -180,6 +188,25 @@ export images="angular-on-nginx django solr schema-spy"
 # The routes for the project
 export routes="angular-on-nginx django solr schema-spy"
 ```
+## Setting up pull secrets
+To get around docker rate limiting, you may need to create an account on docker.io or use artifactory. In order to do this, you will need to set up
```

Comment:
Nit.: rather than calling out Artifactory, what about wording this as `another docker image registry (e.g.: Artifactory)`?

Reply:
Agreed, keep the references generic.  Only provide specific implementations as examples.